### ISSUE TITLE
List of documentation related issues

### ISSUE THREAD

**Issue by Burrito78** [dosbox-staging/dosbox-staging#2479]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### What Operating System are you using?

Windows 11

### Is your feature request related to a problem? Please describe.

d

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by Grounded0** [comment]
Hey you can mark them like this:
- #1672

**Comment by johnnovak** [comment]
https://github.com/dosbox-staging/dosbox-staging/issues/2590#issuecomment-1585733302

**Comment by Grounded0** [comment]
Can be closed?

**Comment by johnnovak** [comment]
> Can be closed?

No, the user manual is not online yet. It will come later this year.
